variables.tf

```diff
@@ -194,3 +199,69 @@ variable "bamboo_install_local_chart" {
   type        = bool
   default     = false
 }
+
+
+################################################################################
+# Jira Settings
+################################################################################
+
+variable "jira_db_major_engine_version" {
+  description = "The Postgres major version to use."
```

Comment:
Better to not point to Postgres here and instead say "The database major version to use."
The reason is we have the only database supported is Postgress and if in future we support another database this variable will apply to different db as well.